README.md

```diff
@@ -4,6 +4,10 @@ Classes for triggering builds of snaps and Ubuntu images through the [Launchpad
 
 ## Usage
 
+### Local development
+
+For local development, it's best to test this module with one of our website projects like [ubuntu.com]([url](https://github.com/canonical-web-and-design/ubuntu.com/)). For more information, follow [this guide (internal only)]([url](https://discourse.canonical.com/t/how-to-run-our-python-modules-for-local-development/308)).
```

Comment:
Seems like the markdown URL notation was wrong:

```suggestion
For local development, it's best to test this module with one of our website projects like [ubuntu.com](https://github.com/canonical-web-and-design/ubuntu.com/). For more information, follow [this guide (internal only)](https://discourse.canonical.com/t/how-to-run-our-python-modules-for-local-development/308).
```

Reply:
I checked this is a VSCode and is rendered correctly but looks like GH is different.

Reply:
Works well on GH now:

<img width="941" alt="image" src="https://user-images.githubusercontent.com/83575/170509160-96c30540-ed54-4698-934e-9c1117e1949f.png">